monitoring/mongodb/dashboard.py

```diff
@@ -0,0 +1,1359 @@
+)
+
+disk_reads = mongodb_timeseries(
+    "Disk Reads Completed",
+    [
+        Target(
+            expr=f'sum by(pod) (rate(container_fs_reads_total{{namespace="{NAMESPACE}", pod=~".*mongodb.*"}}[$__rate_interval]))',
+            legendFormat="{{pod}}",
+        )
+    ],
+    unit=UNITS.OPS_PER_SEC,
+)
+
+disk_writes = mongodb_timeseries(
+    "Disk Writes Completed",
+    [
+        Target(
+            expr=f'sum by(pod) (rate(container_fs_writes_total{{namespace="{NAMESPACE}", pod=~".*mongodb.*"}}[$__rate_interval]))',
+            legendFormat="{{pod}}",
+        )
+    ],
+    unit=UNITS.OPS_PER_SEC,
+)
+
+# Oplog Panels
+oplog_recovery = mongodb_timeseries(
+    "Oplog Recovery Window",
+    [
+        Target(
+            expr=f'mongodb_mongod_replset_oplog_head_timestamp{{job=~"{Jobs.shard_pattern}"}}-mongodb_mongod_replset_oplog_tail_timestamp{{job=~"{Jobs.shard_pattern}"}}',
+            legendFormat="{{ pod }}",
+        )
+    ],
+    unit=UNITS.SECONDS,
+    lineWidth=2,
+    fillOpacity=20,
+    showPoints="never",
+    description="Timespan window between newest and the oldest op in the Oplog collection.",
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["mean", "max", "min"],
+)
+
+avg_op_latency = mongodb_timeseries(
+    "Average Operation Latency",
+    [
+        Target(
+            expr=f'sum by(type) (\n  rate(mongodb_mongod_op_latencies_latency_total{{job=~"{Jobs.shard_pattern}"}}[$__rate_interval])\n) /\nsum by(type) (\n  rate(mongodb_mongod_op_latencies_ops_total{{job=~"{Jobs.shard_pattern}"}}[$__rate_interval])\n)',
+            legendFormat="{{type}}",
+        )
+    ],
+    unit="µs",
+    thresholds=THRESHOLDS["latency"],
+    lineInterpolation="smooth",
+    fillOpacity=20,
+    showPoints="auto",
+    description="Average latency of operations (classified by read, write, or (other) command)",
+)
+
+replication_lag = mongodb_timeseries(
+    "Replication Lag",
+    [
+        Target(
+            expr=MongoDBMetrics.replication_lag(),
+            legendFormat="{{pod}} <- {{name}}",
+        )
+    ],
+    unit=UNITS.SECONDS,
+    thresholds=THRESHOLDS["replication_lag"],
+    lineInterpolation="smooth",
+    lineWidth=2,
+    fillOpacity=20,
+    showPoints="always",
+    decimals=2,
+    valueMin=0,
+    description="MongoDB replication lag occurs when the secondary node cannot replicate data fast enough to keep up with the rate that data is being written to the primary node.",
+)
+
+flow_control = mongodb_timeseries(
+    "Flow Control",
+    [
+        Target(
+            expr=f'max(rate(mongodb_ss_flowControl_isLaggedCount{{job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}}[$__rate_interval]))',
+            legendFormat="fc_count",
+        ),
+        Target(
+            expr=f'max(rate(mongodb_ss_flowControl_isLaggedTimeMicros{{job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}}[$__rate_interval]))',
+            legendFormat="fc_time",
+        ),
+    ],
+    unit="µs",
```

Comment:
`fc_count` unit is not mirosecond? So we should have 2 scales in this panel, one for each serie.